app/text_buffer.py

```diff
@@ -172,9 +170,31 @@ def cursorLeft(self):
     self.selectionNone()
     self.cursorMoveLeft()
 
+  def getCursorMove(self, rowDelta, colDelta):
+    return self.getCursorMoveAndMark(rowDelta, colDelta, 0, 0, 0)
+
   def cursorMove(self, rowDelta, colDelta):
     self.cursorMoveAndMark(rowDelta, colDelta, 0, 0, 0)
 
+  def getCursorMoveAndMark(self, rowDelta, colDelta, markRowDelta,
```

Comment:
It looks like much of the contents of cursorMoveAndMark() can be replaced with a a call to getCursorMoveAndMark(). Does that work?